src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp

```diff
@@ -155,7 +155,10 @@
     <br/>
 
     <kra-pd:keyPersons/>
-
+    <div id = "workarea">
+        <kra-pd:proposalDevelopmentQuestionnaireAnswersFiltered bean = "${KualiForm.questionnaireHelper}" property = "questionnaireHelper"/>
+        <kra-pd:proposalDevelopmentQuestionnaireAnswersFiltered bean = "${KualiForm.s2sQuestionnaireHelper}" property = "s2sQuestionnaireHelper" forceNonTransparent="${forceTabNonTransparent}"/>
```

Comment:
No need to display the S2s questions + answers here...

Reply:
No need to display the S2s questions + answers here...